README.md

```diff
@@ -1,2 +1,35 @@
-# docker_swarm_autoupdater
-Updates Docker Swarm Service images automatically
+# Docker Swarm Autoupdater
+
+Updates Docker Swarm Service images automatically when calling a webhook.
+Can be used for deploying new versions of your application in your docker swarm cluster.
+
+## Usage
+
+Create a service for the autoupdater and mount the docker socket to the container.
+
+```yaml
+version: '3.7'
+services:
+    autoupdater:
+      image: woolfg/docker-swarm-autoupdater:latest
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+      environment:
+        PORT: 80
+        HOOK: /update-PUT_YOUR_RANDOM_STRING_HERE
+      deploy:
+        placement:
+          constraints:
+            - node.role == manager
```

Comment:
Is this constraint required?

Reply:
yup, has to run on a manager node of docker swarm